example/libbpftime_example/main.cpp

```diff
@@ -0,0 +1,29 @@
+#include "bpftime_shm.hpp"
+#include <iostream>
+
+int main() {
+    // Initialize global shared memory with SHM_CREATE_OR_OPEN option
+    bpftime_initialize_global_shm(bpftime::shm_open_type::SHM_CREATE_OR_OPEN);
+
+    // Create a shared memory segment with a name and size
+    const char* shmName = "example_shm";
+    const size_t shmSize = 1024; // 1KB
+    boost::interprocess::managed_shared_memory shm(boost::interprocess::create_only, shmName, shmSize);
+
+    // Allocate a string in the shared memory
+    typedef boost::interprocess::allocator<char, boost::interprocess::managed_shared_memory::segment_manager> CharAllocator;
```

Comment:
I think it would be better to use a more meaningful example, maybe you can try something like dump the json content of the shared memory? You can find the code in tools/bpftimetool

Reply:
I have added an example which loads some random insn from json to global memory and then back to a different json.